hyrisecockpit/frontend/src/views/WorkloadMonitoring.vue

```diff
@@ -30,6 +34,7 @@ import { useSelectionHandling } from "@/meta/selection";
 import StatusWarning from "@/components/alerts/StatusWarning.vue";
 import SelectionList from "@/components/selection/SelectionList.vue";
 import MetricsComparisonTable from "@/components/container/MetricsComparisonTable.vue";
+import DatabaseSystemDetails from "../components/details/DatabaseSystemDetails.vue";
```

Comment:
```suggestion
import DatabaseSystemDetails from "@/components/details/DatabaseSystemDetails.vue";
```

This makes life easier, because it is a dynamic import from root directory